src/util/jira-utils.ts

```diff
@@ -43,6 +43,11 @@ export const getJiraAuthor = (...authors: (Author | undefined)[]): JiraAuthor =>
 	};
 };
 
+export const limitCommitMessage = (message: string ): string => {
+	const MAX_COMMIT_MESSAGE_LENGTH = 1024;
+	return message.substring(0, MAX_COMMIT_MESSAGE_LENGTH);
```

Comment:
I believe i can omit the `|| ""` in the return now since type string is required?

Reply:
the type is there, but doesn't mean the data will always be not undefined in runtime.  I'd probably set the function definition as `(message = "")` so we have a default if nothing is specified :)